.circleci/config.yml

```diff
@@ -47,56 +120,63 @@ jobs:
       - run:
           name: Unit tests
-          command: yarn run test
+          environment:
+            E2E_TEST_IDS: '' # Empty string equals to false in JS
+          command: yarn run test --maxWorkers $TOTAL_CPUS
       - run:
           name: Ruby version
           command: ruby --version
       - run:
           name: Fastlane Prepare Android Keystore
-          command: bundle exec fastlane android keystore
+          command: bundle exec fastlane keystore
           working_directory: android
       - restore_cache:
-          key: gradle-{{ checksum "android/build.gradle" }}-{{ checksum "android/app/build.gradle" }}-{{ checksum "android/settings.gradle" }}
+          keys:
+            - 1-gradle-{{ checksum "android/build.gradle" }}-{{ checksum "android/app/build.gradle" }}-{{ checksum "android/settings.gradle" }}
+            - 1-gradle-
       - run:
           name: Download Dependencies
-          command: bundle exec fastlane android dependencies
+          command: bundle exec fastlane dependencies
           working_directory: android
       - save_cache:
           paths:
             - ~/.gradle
-          key: gradle-{{ checksum "android/build.gradle" }}-{{ checksum "android/app/build.gradle" }}-{{ checksum "android/settings.gradle" }}
+          key: 1-gradle-{{ checksum "android/build.gradle" }}-{{ checksum "android/app/build.gradle" }}-{{ checksum "android/settings.gradle" }}
       - run:
-          name: Fastlane Release Build
-          command: bundle exec fastlane android release
+          name: Fastlane Build
+          command: bundle exec fastlane build
           working_directory: android
-#      - run:
-#          name: Fastlane Browserstack Upload Live
-#          command: bundle exec fastlane android browserstack_upload_live
-#          working_directory: android
-      - run:
-          name: Fastlane E2E Tests on Browserstack
-          command: bundle exec fastlane android browserstack_e2e_tests
-          working_directory: android
-#      - run:
-#          name: Fastlane Play Store Upload
-#          command: bundle exec fastlane android playstore_upload
-#          working_directory: android
       - store_artifacts:
-          path: android/app/build/outputs/apk
+          path: android/app/build/outputs/apk/release/
+      - persist_to_workspace:
+          root: android/app/build/generated/sourcemaps/react/release/
+          paths:
+            - index.android.bundle.map
+      - persist_to_workspace:
+          root: android/app/build/outputs/apk/release/
+          paths:
+            - app-release.apk
 
   build_ios:
+    parameters:
+      e2e_build:
+        description: Whether to create a build for E2E testing
+        type: boolean
+        default: true
     macos:
       xcode: 11.2.0
     environment:
       FL_OUTPUT_DIR: output
-      TOTAL_CPUS: 4 # For class ios medium
+      FASTLANE_SKIP_UPDATE_CHECK: true
```

Comment:
Why do we want to skip this only on iOS?

Reply:
Skipping it everywhere